resonances/project_DV0reso.py

```diff
@@ -99,6 +99,9 @@ def project(config, trigger, pdg_d, pdg_v0):
 
     sel_string = sel_string_d + " and " + sel_string_v0
 
+    cent_min = cfg['selections'][pdg_reso][trigger]['centrality'][0]
+    cent_max = cfg['selections'][pdg_reso][trigger]['centrality'][1]
+    df_all.query(f"{cent_min} < percentile_V0M < {cent_max}")
```

Comment:
```suggestion
    df_all.query(f"{cent_min} < percentile_V0M < {cent_max}", inplace=True)
```
Otherwise the selection has no effect

Reply:
Right, good catch! 
I have fixed it together with the addition of the computation of the normalisation